poker_solver/server/src/game_handler.rs

```diff
@@ -186,7 +187,7 @@ impl GameRunner {
 
         if let Some(player_fold) = hand_state.player_folded() {
             // player folded
-            stacks[usize::from(player_fold) - 1] += pot;
+            stacks[usize::from(player_fold) ] += pot;
```

Comment:
we want to give the pot to the play who did not fold.  This currently assigns the pot to the play who folded which is incorrect